utils/utils.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+// Parse environment variable and return boolean representation of string, or default value if environment variable is unset
+func GetBoolAsStringEnvVar(env string, defaultVal bool) (bool, error) {
+	if val, ok := os.LookupEnv(env); ok {
+		parsedVal, err := strconv.ParseBool(val)
+		if err == nil {
+			return parsedVal, nil
+		}
+		// Environment flag variable exist but with unacceptable format
+		return defaultVal, fmt.Errorf("Environmental flag set but with unacceptable format: %s", err.Error())
+	}
+	// Environment flag variable not exist
+	return defaultVal, nil
+}
+
+// Retrieves the value of specified environment variable, set it with defualt variable if not exist
```

Comment:
nit: "If environment variable is set, return set value, otherwise return default value"